classes/Village.php

```diff
@@ -4,6 +4,78 @@ class Village {
     public System $system;
     public TravelCoords $coords;
 
+    public static array $VillageRep = [
+        1 => [
+            'title' => 'New Villager',
+            'outlaw_title' => 'Thief',
+            'min_rep' => 0,
+        ],
+        2 => [
+            'title' => 'Villager',
+            'outlaw_title' => 'Outlaw',
+            'min_rep' => 100,
+        ],
+        3 => [
+            'title' => 'Well-Known Villager',
+            'outlaw_title' => 'Infamous',
+            'min_rep' => 150,
+        ],
+        4 => [
+            'title' => 'Respected Villager',
+            'outlaw_title' => 'Rogue',
+            'min_rep' => 300,
+        ],
+        5 => [
+            'title' => 'Shinobi',
+            'outlaw_title' => 'Elite Rogue',
+            'min_rep' => 500,
+        ],
+        6 => [
+            'title' => 'Well-known Shinobi',
+            'outlaw_title' => 'Legendary Rogue',
+            'min_rep' => 700,
+        ],
+        7 => [
+            'title' => 'Respected Shinobi',
+            'outlaw_title' => 'Assassin',
+            'min_rep' => 1000,
+        ],
+        8 => [
+            'title' => 'Elite Shinobi',
+            'outlaw_title' => 'Elite Assassin',
+            'min_rep' => 1500,
+        ],
+        9 => [
+            'title' => 'Legendary Shinobi',
+            'outlaw_title' => 'Legendary Assassin',
+            'min_rep' => 3000
+        ],
+        1337 => [
+            'title' => 'Mythos',
+            'outlaw_title' => 'Mythos',
+            'min_rep' => 2147483647
+        ]
+    ];
+    public static array $VillageRenown = [
+
+    ];
+    const ARENA_MISSION_CD = 300;
+    const WEEKLY_REP_CAP = 150;
+    const MISSION_GAINS = [
+        Mission::RANK_D => 1,
+        Mission::RANK_C => 1,
+        Mission::RANK_B => 2,
+        Mission::RANK_A => 3,
+        Mission::RANK_S => 4
+    ];
+    const DAILY_TASK = [
+        DailyTask::DIFFICULTY_EASY => 1,
+        DailyTask::DIFFICULTY_MEDIUM => 2,
+        DailyTask::DIFFICULTY_HARD => 3,
+    ];
+    const PVP_REP = 3;
+    const DECAY = 0.95;
```

Comment:
Clarify here (in comment is fine) what this means - 0.95 rep per day? 5% per day? per week? month?